parm/soca/jcb-base.yaml.j2

```
@@ -72,6 +72,7 @@ observations: all_observations
 
 marine_obsdataroot_path: '{{COMIN_OBS}}'
 marine_obsdatastats_path: '{{COMOUT_OCEAN_ANALYSIS}}/diags'
+marine_obsdatastatsyaml_path: '{{COMOUT_OCEAN_ANALYSIS}}/yaml'
```

Look at this more closely, there should be a COMOUT_CONF that YAMLs should be saved to to be consistent with other components

---

Maybe a job for another PR given the time frame. Make an issue?

---

https://github.com/NOAA-EMC/GDASApp/issues/1749

---

I mean, if I understand correctly all I'm suggesting is:
```suggestion
marine_obsdatastatsyaml_path: '{{COMOUT_CONF}}'
```

---

Yeah, I guess I had a broader vision of doing this across all domains

---

already done for snow and aerosol, maybe already done for atm?

---

atmanl saves to COMOUT_ATMOS_ANALYSIS https://github.com/NOAA-EMC/global-workflow/blob/a15a6db85418c5c71e98a7ce59f5ed26afc5fd6e/ush/python/pygfs/task/atm_analysis.py#L234 but if you're doing aero and snow already, I'll just change it for marine. I forgot that atmanl finalize code isn't yaml-ized yet, so I'll the COMOUT_CONF there later

---

Alright, I just made everything go to COMOUT_CONF, atmanl and atmensanl include. Will retest